src/AuthenticatedAPIClient/axiosConfig.js

```diff
@@ -94,13 +95,20 @@ function applyAxiosInterceptors(authenticatedAPIClient) {
     });
   }
 
-  // Redirect to the logout page if an unauthorized API response was received.
+  // Log errors and info for unauthorized API responses
   function handleUnauthorizedAPIResponse(error) {
-    const errorStatus = error && error.response && error.response.status;
-    if (errorStatus === 401 || errorStatus === 403) {
+    const response = error && error.response;
+    const errorStatus = response && response.status;
+    const requestUrl = response && response.config && response.config.url;
+    const requestIsTokenRefresh = requestUrl === authenticatedAPIClient.refreshAccessTokenEndpoint;
+
+    if (errorStatus === 401 && !requestIsTokenRefresh) {
```

Comment:
It doesn't seem like we should be logging anything here if we aren't handling the error.